test/modules/post/test/file.rb

```diff
@@ -292,7 +292,7 @@ def cleanup
 
   def make_symlink(target, symlink)
     if session.platform == 'windows'
-      cmd_exec("cmd.exe /c mklink #{directory?(target) ? '/D ' : ''}#{symlink} #{target}")
+      cmd_exec("cmd.exe", "/c mklink #{directory?(target) ? '/D ' : ''}#{symlink} #{target}")
```

Comment:
Before this change - we can see in procmon that it's attempting to find the `cmd.exe /c mklink ...` etc on the user's filepath

<img width="1335" alt="image" src="https://github.com/rapid7/metasploit-framework/assets/60357436/0a8d9987-21f0-4c47-b1a6-2095fe520866">

Long term - I think we'll either want to holistically fix this edgecase in the python meterpreter / or update modules to send program name and arguments separately / add a rubocop rule to catch this edgecase